modules/rosa-create-objects.adoc

```diff
@@ -546,4 +597,53 @@ $ rosa create ocm-role [flags]
 
 |===
 
-For more information about the ocm-roles and the roles created with the `rosa create ocm-roles`, see the Additional resources section.
\ No newline at end of file
+For more information about the OCM role created with the `rosa create ocm-role` command, see _Account-wide IAM role and policy reference_.
+
+[id="rosa-create-user-role_{context}"]
+== create user-role
+
+Create the needed user-role resources for your cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa create user-role [flags]
+----
+
+.Flags
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--debug
+|Enable debug mode.
+
+|-i, --interactive
+|Enable interactive mode.
+
+|-m, --mode string
+|How to perform the operation. Valid options are:
+    auto: Resource changes will be automatic applied using the current AWS account
+    manual: Commands necessary to modify AWS resources will be output to be run manually
+
+|--path string
+|The ARN path for the user role and policies.
+
+|--permissions-boundary string
+|The ARN of the policy that is used to set the permissions boundary for the user role.
+
+|--prefix string
+|User-defined prefix for all generated AWS resources (default "ManagedOpenShift")
+
+|--profile string
+|Use a specific AWS profile from your credential file.
+
+|--region string
+|Use a specific AWS region, overriding the AWS_REGION environment variable.
+
+|-y, --yes
+|Automatically answer yes to confirm operation.
+
+|===
+
+For more information about the user role created with the `rosa create user-roles` command, see _Understanding AWS account association_.
```

Comment:
The command at the top is `rosa create user-role` singular, should it be that way here too?

Reply:
Good catch! Resolved.